index.bs

```diff
@@ -1099,18 +1099,25 @@ optional |expires|,
 |sameSite|,
 run the following steps:
 
+1. If |name|, |value|, |domain|, or |path| contain U+003B (`;`), any [=C0 control=] character, or U+007F, then return failure.
 1. If |name|'s [=string/length=] is 0 and |value| contains U+003D (`=`), then return failure.
 1. If |name|'s [=string/length=] is 0 and |value|'s [=string/length=] is 0, then return failure.
+1. Determine whether |name| and |value| together exceed cookie size limits using these steps:
+    1. [=UTF-8 encode=] |name| and then compute its [=byte sequence=] [=byte sequence/length=].
```

Comment:
It's unclear if the intent is to mutate _name_ here from being a string to a byte sequence, or to make an anonymous temporary. (Pretending this is code.)

Since _name_ and _value_ are used below with a sloppy (I wrote it!) invocation of encode, how about redoing this as something like:

1. Let _encodedName_ be the result of UTF-8 encoding _name_.
2. Let _encodedValue_ be the result of UTF-8 encoding _value_.
3. If the byte sequence length of encodedName plus the byte sequence length of encodedValue is greater than 4096, then return failure.